common/graphicalculturetypes/wc_graphicalculturetype.txt

```diff
@@ -493,4 +493,13 @@ nightelffelgfx = {
 }
 highbornefelgfx = {
 	skin_tone = 0
+}
+nightelfundeadgfx = {
```

Comment:
To make race traits work correct you need to add it in `has_nightelf_gfx_trigger` and `has_undead_gfx_trigger`.